src/pages/workspace/WorkspaceInviteMessagePage.js

```diff
@@ -129,6 +127,17 @@ class WorkspaceInviteMessagePage extends React.Component {
         Link.openExternalLink(CONST.PRIVACY_URL);
     }
 
+    focusWelcomeMessageInput() {
+        this.focusTimeout = setTimeout(() => {
+            this.welcomeMessageInputRef.focus();
+            // Below condition is needed for web, desktop and mweb only, for native cursor is set at end by default.
+            if (this.welcomeMessageInputRef.value && this.welcomeMessageInputRef.setSelectionRange) {
+                const length = this.welcomeMessageInputRef.value.length;
+                this.welcomeMessageInputRef.setSelectionRange(length, length);
+            }
+        }, CONST.ANIMATED_TRANSITION);
+    }
```

Comment:
I don't think we need a timeout all the time. This is only needed at mount but not at page refocus

Reply:
nvm, actually we do since we have animation in that case as well.